IdentityService/Config.cs

```diff
@@ -1,40 +1,35 @@
-﻿using Duende.IdentityServer;
-using Duende.IdentityServer.Models;
-
-namespace IdentityService
+﻿namespace IdentityService
 {
     public static class Config
     {
         public static IEnumerable<IdentityResource> IdentityResources =>
-            new[]
+        [
+            new IdentityResources.OpenId(),
+            new IdentityResources.Profile(),
+            new()
             {
-                new IdentityResources.OpenId(),
-                new IdentityResources.Profile(),
-                new IdentityResource
-                {
-                    Name = "role",
-                    UserClaims = new List<string> {"role"}
-                }
-            };
+                Name = "role",
```

Comment:
I think you should keep all those strings like claims, name, api scopes etc... in appsettings